README.md

```diff
@@ -1,142 +1,41 @@
-# Spectral Recovery Tool 
+# (pre-release) spectral-recovery: spectral recovery analysis of forested ecosystems
 
-### Installation and Environment
+[![PyPI version](https://badge.fury.io/py/spectral-recovery.svg)](https://badge.fury.io/py/spectral-recovery) [![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/PEOPLE-ER/Spectral-Recovery/main?labpath=docs%2Fnotebooks%2F)
 
-First, clone the repository
+---
+Github: [https://github.com/PEOPLE-ER/Spectral-Recovery/](https://github.com/PEOPLE-ER/Spectral-Recovery/)
 
-```{bash}
-git clone https://github.com/PEOPLE-ER/Vegetation-Recovery-Assessment.git
-cd Vegetation-Recovery-Assessment/
-```
-
-#### Installing from Wheel (temp. should be removed when package is on PyPi)
-
-Build the package and install wheel.
-
-```{bash}
-python3 -m build
-pip3 install dist/spectral_recovery-0.1-py3-none-any.whl
-```
-The `spectral_recovery` package is now available through import statements and a CLI. See the "Using the Spectral Recovery Tool" notebook for example usage.
-
-#### Installing for Development Mode
-
-To download and perform/test development tasks within the project, enter development mode using an "editable install".
-
-```{bash}
-python -m venv .venv
-pip install --editable .
-```
+Documentation: [https://people-er.github.io/Spectral-Recovery/](https://people-er.github.io/Spectral-Recovery/)
 
-Now the `spectral_recovery` package is accessible as if it was installed in `.venv` (WIP: see "Using the Spectral Recovery Tool" notebook for usage examples). This installation lets you quickly develop the package without building new distributions. 
+PyPi: [https://pypi.org/project/spectral-recovery/](https://pypi.org/project/spectral-recovery/)
 
-When done with a development task, you can simply uninstall the package as you normally would using pip, `pip uninstall spectral_recovery`.
+Interactive Notebooks: [https://mybinder.org/v2/gh/PEOPLE-ER/Spectral-Recovery/main?labpath=docs%2Fnotebooks%2F](https://mybinder.org/v2/gh/PEOPLE-ER/Spectral-Recovery/main?labpath=docs%2Fnotebooks%2F)
```

Comment:
I tried the link that the Bindr notebook didn't load. Will this be fixed by merging it to main?

![image](https://github.com/PEOPLE-ER/Spectral-Recovery/assets/59990244/ebbfc8ec-c221-4f8d-8ae9-dcf273189c41)

Reply:
Yeah, it will resolve once merged into main (not the best PR strategy... but not sure how else to get it going). Does the link I provided in the PR comment work for you? That's the same image but directed at the dev branch!